src/helpers/searchKeyPairStringToArray.ts

```diff
@@ -16,7 +17,7 @@ export const searchKeyPairStringToArray = (
     const values = paramArr
       .splice(1, paramArr.length - 1)
       .reduce((paramRes, val) => {
-        const [key, value] = val.split(":");
+        const [key, value] = val.split(URL_KEY_PAIR_SEPARATOR_CHAR);
```

Comment:
`URL_KEY_PAIR_SEPARATOR_CHAR` is set to `>`, but the previously used value is `:`.

I think `URL_KEY_PAIR_SEPARATOR_CHAR` should be `:` right? I'm looking at the example in the comment on line 4. If not lets update the example.

Reply:
Thanks, will update the comment!

Reply:
@JakeUrban or @quietbits what are the steps to test this locally? I know I need a local instance running but is there a tutorial somewhere that can help me do that? I'm curious to test this although I KNOW @JakeUrban has given it the 💯 approval!

Reply:
You would need to have an anchor running locally, its probably easier for one of us to demo this to you.